AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -22,6 +22,7 @@
 
 static const NSUInteger kASCollectionViewAnimationNone = UITableViewRowAnimationNone;
 static const ASSizeRange kInvalidSizeRange = {CGSizeZero, CGSizeZero};
+static NSString * const reuseIdentifier = @"_ASCollectionViewCell";
```

Comment:
We should name this with a leading "k" or at least a capital letter to signify its special status as a constant.